### ISSUE TITLE
.oxt file not found

### ISSUE THREAD

**Issue by Sc2Marines** [johnwalz97/LibreOfficeAICopilot#2]
I have DL the project , but I could not find the ```libreoffice-ai-copilot.oxt``` file, maybe missing or readme not up to date.

**Comment by watchmann** [comment]
Also could not find the .oxt fie on the LO extensions website?

**Comment by Sc2Marines** [comment]
Yes, I can't find the extension on the website, I have tried with keyword AI, copilot, ai-copilot.

**Comment by balisujohn** [comment]
to make the oxt, you'll want to zip the contents of the libre_copilot folder from inside the folder, then rename the zip file to have an `.oxt` extension. (not 100% percent sure that will work, but I expect it will based on project structure).

**Comment by Loui1979** [comment]
Anyone get this to work?  Did zipi and extract work?
